### DISCUSSION TITLE
Doc for image property

### DISCUSSION THREAD

**Discussion by leorouxx** [jsvine/pdfplumber#1172] [Show and tell]
hello, I have noticed that the README contains an empty field when it comes to describe the "image" property. I don't know if this is relevant, but I have  done the following : 

#### `image` properties

| Property | Description |
|----------|-------------|
|`page_number`| Page number on which the image was found.|
|`height`| Height of the image.|
|`width`| Width of the image.|
|`x0`| Distance of left side of the image from left side of page.|
|`x1`| Distance of right side of the image from left side of page.|
|`y0`| Distance of bottom of the image from bottom of page.|
|`y1`| Distance of top of the image from bottom of page.|
|`top`| Distance of top of the image from top of page.|
|`bottom`| Distance of bottom of the image from top of page.|
|`doctop`| Distance of top of rectangle from top of document.|
|`mcid`| The [marked content](https://ghostscript.com/~robin/pdf_reference17.pdf#page=850) section ID for this image if any (otherwise `None`). *Experimental attribute.*|
|`tag`| The [marked content](https://ghostscript.com/~robin/pdf_reference17.pdf#page=850) section tag for this image if any (otherwise `None`). *Experimental attribute.*|
|`srcsize`| Original Dimension of the image, in pixel (tuple (width, height)).|
|`linewidth`| Thickness of line.|
|`stream`| Pixel values of the image.|
|`imagemask`| #TODO|
|`bits`| Encoding of the image (8 for 0-255 range)|
|`colorspace`| Color domain of the image (e.g. RGB)|
|`object_type`| "image"|

As you can see, I lack the 'imagemask' value, because I could find PDF where this value is not None, and from this [documentation](https://ghostscript.com/~robin/pdf_reference17.pdf#page=339), it seems that it would be a stream-like object. I'm not sure so if any of you actually know what it is I would love to know as well.
Also, please consider giving your feedback on fields you consider not well filled.

Thanks.

**Comment by jsvine** [comment]
Many thanks for this suggestion, @leorouxx. I've added that documentation — with just a few edits of your proposal, plus credit to you — here: https://github.com/jsvine/pdfplumber/commit/937702da25c52ef3df658bc832268c3cf8a6f3d2
